src/getFilenameValidationData.test.ts

```diff
@@ -1,5 +1,5 @@
 import { getFilenameValidationData } from './getFilenameValidationData';
-import { filenames } from '../fixtures';
+import { filenames } from './fixtures';
```

Comment:
@wholesomedev When you compile a TS project, and even if you configure `tsconfig.json` to only include a sub-folder, TS will still compile files outside that folder if they are imported somewhere in the project. Because `fixtures` was outside `src` and was required in this file, TS would copy the directory structure to `bin`, starting with the outermost level, so we would get:

```
bin
-- fixtures.js
-- src
   -- (files)
```

On the other hand, when `fixtures` is inside `src`, the top-level directory for TypeScript will be `src`, and we get:

```
bin
--files
--fixtures.js
```